test/runtests.jl

```diff
@@ -40,7 +40,6 @@ for g in groups
         include("dimensions.jl")
         include("indexing.jl")
         include("ranges.jl")
-        include("setup.jl")
```

Comment:
what was this file?

Reply:
It's the first file included in that list. We were just including it twice.